### PR TITLE
Updating the entity model relationship diagram closes #11

### LINKED ISSUE #11: Update Capture entity with Approval/Disapproval property
DOD: When the approval/disapproval property is added, approved by the team, and merged into the develop branch.

### LINKED ISSUE #14: Update the entity model relationship diagram
DOD: When the new EMR Diagram is complete and approved by the client.

### PR DESCRIPTION
**Pull request by weberwilliamb**
Team - I have made all the updates to the Entity model and connected each accordingly. Please review.

### CODE REVIEW

**Review by MaxIgnatyev — COMMENTED** [review]
Looks like you simplified it which is good, relationships are more clear.

**Comment by Sara-webaplication** [comment]
I approved it, thank you.

File: photo-points/Models/Capture.cs [hunk 1]
```diff
@@ -12,6 +12,9 @@ public class Capture
     public DateTime captureDate { get; set; }
     public long photoPointID { get; set; }
     public long userID { get; set; }
+    public User user { get; set; }
+    public PhotoPoint PhotoPoint { get; set; }
+
```
Comment (Sara-webaplication) [inline photo-points/Models/Capture.cs:18]: looks ok to me

**Review by Sara-webaplication — APPROVED** [review]

**Review by sruble — COMMENTED** [review]
looks good

**Comment by Bansenauer-Cascadia** [comment]
Max and Sim - you need to select that you approve (or request changes to) a pull request rather than just comment that it "looks good".

Use the "Add your review" button and then click "Approve" option (or a different one if you see changes needed.)

**Review by RoEric — APPROVED** [review]
Looks like the models appropriately capture what we talked about in our client meeting now

**Review by sruble — APPROVED** [review]
Looks Good

**Review by MaxIgnatyev — APPROVED** [review]

**Pull request merged**